modules/wallpapers/ReadMe.txt

```diff
@@ -24,4 +26,4 @@ To include the module in your build folder, run the configure script with the fl
 
 # For UNIX users
 
-    ./configure.sh -DENABLE_WALLPAPERS=1
\ No newline at end of file
+    ./configure.sh -DENABLE_WALLPAPERS=1
```

Comment:
Whitespace change appeared again.

Reply:
:/ this is getting annoying

Reply:
Just don't use GitHub text editor. It's better to use Notepad++ and commit locally on your PC, then push the changes.

Reply:
strange; this happened once i added a newline.. is it something to do with LF/CRLF? hm..

Reply:
also i have been using notepad++, thats the weird thing

Reply:
ok i fixed it by removing the newline? strange..

Reply:
ok im not sure what is causing the line removal but it only happens when a newline is appended

Reply:
Adding newlines to the ends of files is perfectly okay, don't worry about it.

Reply:
alright then